noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1503,130 +1869,14 @@
-                        COALESCE(r.val, 0) number_of_record_cross_references
-                    FROM arkivstruktur.arkivdel s
-                    LEFT JOIN
-                    (
-                        SELECT ks._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.kryssreferanse cr
-                        JOIN arkivstruktur.klasse k ON k.systemid = cr._ref_klasse
-                        JOIN arkivstruktur.klassifikasjonssystem ks ON ks.systemid = k._ref_klassifikasjonssystem
-                        GROUP BY ks._ref_arkivdel
-                    ) k ON k.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT f._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.kryssreferanse cr
-                        JOIN arkivstruktur.mappe f ON f.systemid =  cr._ref_mappe
-                        GROUP BY f._ref_arkivdel
-                    ) f ON f.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.kryssreferanse cr
-                        JOIN arkivstruktur.registrering r ON r.systemid = cr._ref_registrering
-                        GROUP BY r._ref_arkivdel
-                    ) r ON r.series_system_id = s.systemid;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Precedent count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT s.systemid series_system_id,
-                        COALESCE(f.val, 0) number_of_file_precedents,
-                        COALESCE(r.val, 0) number_of_record_precedents
-                    FROM arkivstruktur.arkivdel s
-                    LEFT JOIN
-                    (
-                        SELECT f._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.presedens p
-                        JOIN arkivstruktur.mappe f ON f.systemid = p._ref_mappe
-                        GROUP BY f._ref_arkivdel
-                    ) f ON f.series_system_id = s.systemid
-                    LEFT JOIN
-                    (
-                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
-                        FROM arkivstruktur.presedens p
-                        JOIN arkivstruktur.registrering r ON r.systemid = p._ref_registrering
-                        GROUP BY r._ref_arkivdel
-                    ) r ON r.series_system_id = s.systemid;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST18">
         <title>Correspondence Party count</title>
+        <description>
+            Provides information about the number of Correspondence Parties grouped
```

Comment:
`Provides information about the number of correspondence parties grouped by series and tests whether any registry entries without correspondence parties exist`